data/forms/ethnicity/ethnicity-category-black-black-british.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "ethnicity-category-black-black-british",
+  "heading": "Which of the following best describes your Black or Black British background?",
+  "steps": [
+    {
+      "fields": [
+        {
+          "as": "radios",
+          "label": "Please Select",
+          "name": "ethnicity-category-black-black-british",
```

Comment:
I think the name for each sub category should be `ethnicity-extended-category` so they are grouped and follows convention used by Hackney.